README.md

```diff
@@ -0,0 +1,94 @@
+# gr-stellarstation: GNURadio blocks for interacting with StellarStation
+
+gr-stellarstation is an out-of-tree GNURadio module that provides blocks for easily integrating with the [StellarStation](https://www.stellarstation.com/) API in GNURadio flowgraphs.
+
+## Installation
+
+### PyBOMBS
+
+PyBOMBs support is currently under construction.
+
+### Source Installation
+
+gr-stellarstation, like most OOT modules, can be built from source using CMake.
+
+First, we need to install gr-stellarstation's sole dependency, [Conan](https://conan.io/). The simplest way to do this is through `pip`:
```

Comment:
Can you specify required major version of Python? (i.e. Python 3 or Python 2)

Reply:
GNURadio only works on Python 2 right now.

Reply:
Although Python 2 has extended maintenance period, moving to Python 3 is encouraged. We should consider default version of Python in users are Python 3.x and give extra notice if the build requires older version, Python 2.